### PR TITLE
Add imports for Bruker PrairieView data

### PR DESCRIPTION
**Pull request by kushalbakshi**

### CODE REVIEW

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @kushalbakshi.  Were you able to populate the PrairieView data through the entire schema?  I believe you will need to also update this list:

```
image_locators = {
    "NIS": get_nd2_files,
    "ScanImage": get_scan_image_files,
    "Scanbox": get_scan_box_files,
}
```

Please update:
- [x] version.py
- [x] changelog.md
- [ ] docs/src/concepts.md to specify that we now support PrairieView

**Comment by kushalbakshi** [comment]
> Thanks @kushalbakshi. Were you able to populate the PrairieView data through the entire schema? I believe you will need to also update this list:
> 
> ```
> image_locators = {
>     "NIS": get_nd2_files,
>     "ScanImage": get_scan_image_files,
>     "Scanbox": get_scan_box_files,
> }
> ```
> 
> Please update:
> 
>     * [x]  version.py
> 
>     * [x]  changelog.md
> 
>     * [x]  docs/src/concepts.md to specify that we now support PrairieView

I only had enough data to populate an entry in the `ScanInfo` table. I now have datasets from both interested labs to try a full test

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.4.3] = 2022-12-14
++ Update - Added file locator functions for Bruker PrairieView data.
```
Comment (kabilar) [inline CHANGELOG.md:6]:
```suggestion
## [0.5.1] - 2022-12-14
+ Add - File locator functions for Bruker PrairieView data. 
```

File: element_calcium_imaging/version.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = "0.4.2"
+__version__ = "0.4.3"
```
Comment (kabilar) [inline element_calcium_imaging/version.py:2]:
```suggestion
__version__ = "0.5.1"
```

File: docs/src/concepts.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ Major features of the Calcium Imaging Element include:
 + Calcium-imaging scanning metadata, also compatible with mesoscale imaging and multi-ROI scanning mode
 + Tables for all processing steps: motion correction, segmentation, cell spatial footprint, fluorescence trace extraction, spike inference and cell classification
 + Store/track/manage different curations of the segmentation results
-+ Ingestion support for data acquired with ScanImage and Scanbox acquisition systems
++ Ingestion support for data acquired with ScanImage, Scanbox, Nikon NIS, and Bruker acquisition systems
```
Comment (kabilar) [inline docs/src/concepts.md:63]:
```suggestion
+ Ingestion support for data acquired with ScanImage, Scanbox, Nikon NIS, and PrairieView acquisition systems
```

File: CHANGELOG.md (hunk 1 above)
Comment (kabilar) [reply CHANGELOG.md:6]: Please add the following to the bottom of the file:
```
[0.5.1]: https://github.com/datajoint/element-calcium-imaging/releases/tag/0.5.1
```

**Comment by kabilar** [comment]
Please pull from upstream to fast forward your branch.

**Review by kabilar — APPROVED** [review]
Thanks @kushalbakshi!

**Pull request merged**